blogpost: Migrating From Brave to Desktop App (#693)
ipfs/ipfs-blog: src/_blog/2024-brave-migration-guide.md

This guide will walk you through the process of moving your IPFS data from Brave to IPFS Desktop, ensuring you don't lose any of your important files, and keep access to IPFS resources in your browser.

## Why Migrate?

- **Imminent Removal:** The IPFS node feature in Brave is being [phased out](https://github.com/brave/brave-browser/issues/37735#issuecomment-2247764368) and will happen once v1.69.122 or later is released ([schedule](https://github.com/brave/brave-browser/wiki/Brave-Release-Schedule#release-channel-dates)). To ensure uninterrupted access to your IPFS data, migration is necessary, especially if you pinned something, or published with IPNS.

> - **Limitations of the Brave Implementation:** The Brave-integrated IPFS node had some drawbacks. The access to WebUI was hidden behind `brave://ipfs-internal`. DNSLink detection was based on HTTP header rather than DNS TXT lookup. Running IPFS node required the Brave browser to be open for content and IPNS announcements to function, and in early days did not even start `ipfs daemon` before `ipfs://` was used for the first time, leading to content from local repository not being provided to IPFS Mainnet peers.

Author: Unsure about this section – lmk if it does not add much value now that its gone anyway, I'm fine with removing it.

> - **Limitations of the Brave Implementation:** The Brave-integrated IPFS node had some drawbacks. The access to WebUI was hidden behind `brave://ipfs-internal`. DNSLink detection was based on HTTP header rather than DNS TXT lookup. Running IPFS node required the Brave browser to be open for content and IPNS announcements to function, and in early days did not even start `ipfs daemon` before `ipfs://` was used for the first time, leading to content from local repository not being provided to IPFS Mainnet peers.

Author: Unsure about mentioning **Limitations of the Brave Implementation:** – lmk if it does not add much value now that its gone anyway, I'm fine with removing it:
```suggestion
```

Reviewer: my pref would be to make the "why" section clearly and solely about the removal, and move the limitations bit to a separate section about "what we learned was hard" or something like that.

Author: @autonome moved "limitations of Brave"  to `## Optional: Adjusting Kubo Configuration` with instruction how to increase cache size. This way it becomes a necessary context why we suggest to user to make some optional configuration changes.

> - **Improved Functionality:** Migrating to a standalone IPFS solution like IPFS Desktop offers several advantages: (1) Automatic security and performance updates without relying on browser updates. (2)  Ability to customize your IPFS node configuration, no vendor-specific overrides. (3) Browser-agnostic background service, allowing your node to run independently of any specific browser. (4) Easy access to your files in WebUI via system status bar icon.

Reviewer 2: ```suggestion
- **Improved Functionality:** Migrating to a standalone IPFS solution like IPFS Desktop offers several advantages: 
  1. Automatic security and performance updates without relying on browser updates.
  2. Ability to customize your IPFS node configuration, no vendor-specific overrides.
  3. Browser-agnostic background service, allowing your node to run independently of any specific browser.
  4. Easy access to your files in WebUI via system status bar icon.
```

### Time Investment

Migrating your IPFS node is a relatively quick process. Most users can complete the transition in 5 to 15 minutes, depending on their familiarity with IPFS and their system configuration.

## Prerequisites

Before we begin, you'll need to install two key components that will replace the functionality that was in Brave with an IPFS stack that will still work in Brave, but also in most other browsers.

> - [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/) is a full node application that runs on your computer, managing your IPFS repository and providing a graphical interface for IPFS operations. Download IPFS Desktop by following the [install guide here](https://docs.ipfs.tech/install/ipfs-desktop/#install-instructions). Choose the appropriate version for your operating system ([Windows](https://docs.ipfs.tech/install/ipfs-desktop/#windows), [macOS](https://docs.ipfs.tech/install/ipfs-desktop/#macos), or [Linux](https://docs.ipfs.tech/install/ipfs-desktop/#ubuntu)) and follow the installation instructions.

Reviewer 2: ```suggestion
- [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/) is a full node application that runs [Kubo](https://github.com/ipfs/kubo/) on your computer, managing your IPFS repository and providing a graphical interface for IPFS operations. Download IPFS Desktop by following the [install guide here](https://docs.ipfs.tech/install/ipfs-desktop/#install-instructions). Choose the appropriate version for your operating system ([Windows](https://docs.ipfs.tech/install/ipfs-desktop/#windows), [macOS](https://docs.ipfs.tech/install/ipfs-desktop/#macos), or [Linux](https://docs.ipfs.tech/install/ipfs-desktop/#ubuntu)) and follow the installation instructions.
```

- [IPFS Companion](https://docs.ipfs.tech/install/ipfs-companion/) is a browser extension that allows you to interact with IPFS content directly from your web browser, load it from your local IPFS node, and keep provisional support for `ipfs://` and `ipns://` in address bar. The easiest way to install IPFS Companion is through your browser's specific [extensions and add-ons store](https://docs.ipfs.tech/install/ipfs-companion/#install).

## Moving the Brave IPFS Repository

> The IPFS repository, often referred to as `$IPFS_PATH` (aka `.ipfs`), contains all your IPFS data, IPNS keys, and PeerID identify of your IPFS node. Brave used the same repository format as Kubo, making migration to IPFS Desktop relatively easy.

Reviewer 2: ```suggestion
The IPFS repository, often referred to as `$IPFS_PATH` (aka `.ipfs`), contains all your IPFS data, IPNS keys, and PeerID identify of your IPFS node. Brave's IPFS Node used the same repository format as Kubo, making migration to IPFS Desktop relatively easy.
```

- Linux: `~/.config/BraveSoftware/Brave-Browser/brave_ipfs`

To conform you've found the right directory, open `brave_ipfs/config` and write down the value of `PeerID`, it will act as unique identifier of your Brave repository.

Now, we'll move this repository to the default location for IPFS Desktop:

- Windows: `%USERPROFILE%/.ipfs`
- macOS and Linux: `~/.ipfs`

Before proceeding, make sure the `.ipfs` does not exist at the destination. If you already had `.ipfs`, shut down IPFS Desktop and rename `.ipfs` to `.ipfs.old` as a precaution to avoid data loss.

Now, move the `brave_ipfs` directory from Brave profile, to the location expected by IPFS Desktop.

You can use the following commands in your terminal or command prompt:

For Windows:

```
IF NOT EXIST "%USERPROFILE%\.ipfs" MOVE "%LOCALAPPDATA%\BraveSoftware\Brave-Browser\User Data\Default\brave_ipfs" "%USERPROFILE%\.ipfs"
```

For macOS:
```
test ! -d ~/.ipfs && mv ~/Library/Application\ Support/BraveSoftware/Brave-Browser/Default/brave_ipfs ~/.ipfs
```

Linux:
```
test ! -d ~/.ipfs && mv ~/.config/BraveSoftware/Brave-Browser/brave_ipfs ~/.ipfs
```

## Starting IPFS Desktop with Migrated IPFS Repository

> Once move is completed, you can confirm it was successful if `.ipfs/config` exists, and includes `PeerID` of your Brave node.

Reviewer 2: ```suggestion
Once move is completed, you can confirm it was successful if `.ipfs/config` exists in your home directory, and includes `PeerID` of your Brave node.
```

- [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/) is a full node application that runs [Kubo](https://github.com/ipfs/kubo/) on your computer, managing your IPFS repository and providing a graphical interface for IPFS operations. Download IPFS Desktop by following the [install guide here](https://docs.ipfs.tech/install/ipfs-desktop/#install-instructions). Choose the appropriate version for your operating system ([Windows](https://docs.ipfs.tech/install/ipfs-desktop/#windows), [macOS](https://docs.ipfs.tech/install/ipfs-desktop/#macos), or [Linux](https://docs.ipfs.tech/install/ipfs-desktop/#ubuntu)) and follow the installation instructions.
- [IPFS Companion](https://docs.ipfs.tech/install/ipfs-companion/) is a browser extension that allows you to interact with IPFS content directly from your web browser, load it from your local IPFS node, and keep provisional support for `ipfs://` and `ipns://` in address bar. The easiest way to install IPFS Companion is through your browser's specific [extensions and add-ons store](https://docs.ipfs.tech/install/ipfs-companion/#install).

## Moving the Brave IPFS Repository

> The IPFS repository, often referred to as `$IPFS_PATH` (aka `.ipfs`), contains all your IPFS data, IPNS keys, and PeerID identify of your IPFS node. Brave's IPFS Node used the same repository format as Kubo, making migration to IPFS Desktop relatively easy.

Reviewer 3: ```suggestion
The IPFS repository, often referred to as `$IPFS_PATH` (aka `~/.ipfs`), contains all your IPFS data, IPNS keys, and PeerID identify of your IPFS node. Brave's IPFS Node used the same repository format as Kubo, making migration to IPFS Desktop relatively easy.
```

If you did not use IPFS Desktop before, you can simply swap `.ipfs` created by IPFS Desktop with the one from your Brave node. This is the simplest way of migrating your node, all data, pins, IPNS keys, addresses and PeerID will remain the same and IPNS publishing will continue working.

First, we need to locate your Brave IPFS repository:

> <!-- TODO: confirm these paths are valid -->

Reviewer 3: is this part confirmed? output for mac for any file/dir with "ipfs" in the name is:

```sh
❯ rg -g '*ipfs*' --files ~/Library/Application\ Support/BraveSoftware
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/lejaflgbgglfaomemffoaappaihfligf/1.0.29/go-ipfs_v0.29.0_darwin-arm64
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/ipfs-logo-on.svg
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/brave-ipfs-logo-off.svg
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/brave-ipfs-logo-on.svg
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/ipfs-logo-off.svg
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/png/ipfs-logo-on_19.png
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/png/ipfs-logo-off_19.png
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/png/ipfs-logo-off_38.png
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/png/ipfs-logo-on_128.png
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/png/ipfs-logo-on_38.png
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/icons/png/ipfs-logo-off_128.png
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/dist/bundles/ipfs.bundle.js
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/dist/popup/browser-action/ipfs-version.js
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/nibjojkomfdiaoajekhjakgkdhaomnch/3.1.0_0/dist/options/forms/ipfs-node-form.js
/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/Default/Extensions/fpeoodllldobpkbkabpblcfaogecpndd/0.12.7_0/brave-ipfs.json

```

Reviewer 3: FYI it looks to be at `/Users/sgtpooki/Library/Application Support/BraveSoftware/Brave-Browser/brave_ipfs/config` on mac

Author: Thank you! I looked at Windows one and here too we don't use `/Default` so I've updated blogpost with updated paths + added examples of fully resolved ones.

- Windows: `%LOCALAPPDATA%\BraveSoftware\Brave-Browser\User Data\Default\brave_ipfs`
- macOS: `~/Library/Application Support/BraveSoftware/Brave-Browser/Default/brave_ipfs`
- Linux: `~/.config/BraveSoftware/Brave-Browser/brave_ipfs`

To conform you've found the right directory, open `brave_ipfs/config` and write down the value of `PeerID`, it will act as unique identifier of your Brave repository.

Now, we'll move this repository to the default location for IPFS Desktop:

- Windows: `%USERPROFILE%/.ipfs`
- macOS and Linux: `~/.ipfs`

> Before proceeding, make sure the `.ipfs` does not exist at the destination. If you already had `.ipfs`, shut down IPFS Desktop and rename `.ipfs` to `.ipfs.old` as a precaution to avoid data loss.

Reviewer 3: ```suggestion
Before proceeding, make sure the `.ipfs` directory does not exist at the destination. If you already had `.ipfs`, shut down IPFS Desktop and rename `.ipfs` to `.ipfs.old` as a precaution to avoid data loss.
```

Alternative is to update `.ipfs/config` and replace all occurences of `45001` with `5001` and `48080` with `8080`. Make sure you do not have anything listening on these ports before you make the change.

## Conclusion

Congratulations! You've successfully migrated your IPFS data from Brave to IPFS Desktop.

If you encountered any challenges during the migration process or need further assistance, please don't hesitate to leave a comment in the thread below. The community is here to help, and your feedback can also assist others who might be going through the same process.

## FAQ

### Is it possible to move `brave_ipfs` to a different location than `.ipfs`?

Yes, but one needs to set `IPFS_PATH` environment variable before running IPFS Desktop to point at the new location.

See [How does IPFS Desktop select the IPFS repo location?](https://github.com/ipfs/ipfs-desktop/?tab=readme-ov-file#how-does-ipfs-desktop-select-the-ipfs-repo-location)

### Where can I find FAQ/Troubleshooting for IPFS Desktop?

See [github.com/ipfs/ipfs-desktop/#faq--troubleshooting](https://github.com/ipfs/ipfs-desktop/?tab=readme-ov-file#faq--troubleshooting)

### Can Kubo be used instead?

> Yes, advanced users who are comfortable with command-line can use [Kubo](https://docs.ipfs.tech/install/command-line/) instead of IPFS Desktop, and run it against a custom `IPFS_PATH` to run a headless daemon, or perform selective manual migration via CLI.

Reviewer 3: could benefit to mention this also comes with ipfs-webui built in?

Author: I don't think it matters for the migration. WebUI does not support CAR import, nor IPNS key import/export, only CLI tools can be used for such low level migration.

---
title: 'Migrating from Brave to IPFS Desktop'
description: 'Complete Guide to Migrating Your IPFS Data'

> date: 2024-08-22

Reviewer 2: TODO: update to actual publishing date

header_image: '/blog-post-placeholder.png'
tags:
- brave
- browsers
- IPFS Desktop
---

> Brave browser users who have been relying on the built-in IPFS node functionality will soon need to find an alternative solution. We could not find a mutually agreeable set of terms, and Brave has [announced](https://github.com/brave/brave-browser/issues/37735) the removal of support for running IPFS nodes on behalf of users. This change was implemented in the latest stable release ([v1.69.153](https://github.com/brave/brave-browser/blob/56f6418ac301a4b015c1188786f6f4497b6ac393/CHANGELOG_DESKTOP.md#169153)) which shipped on Aug 22nd. While this change may be disappointing for some, it presents an opportunity to adopt a more robust and flexible IPFS setup.

Reviewer 2: ```suggestion
In [2021](https://brave.com/blog/ipfs-support/), IPFS maintainers worked with the Brave team to add native support for IPFS in the Brave Browser. This was the first deep integration of an IPFS node in a browser.

After over three years, the Brave team [decided](https://github.com/brave/brave-browser/issues/37735) remove support for running IPFS node as we could not find a mutually agreeable set of terms to make this integration sustainable. The removal was implemented in the latest stable release ([v1.69.153](https://github.com/brave/brave-browser/blob/56f6418ac301a4b015c1188786f6f4497b6ac393/CHANGELOG_DESKTOP.md#169153)) which shipped on Aug 22nd. 

While this change may be disappointing for some, it presents an opportunity to adopt a more robust and flexible IPFS setup.
```

> This guide will walk you through the process of moving your IPFS data from Brave to IPFS Desktop, ensuring you don't lose any of your important files, and keep access to IPFS resources in your browser.

Reviewer 2: ```suggestion
This guide will walk you through the process of moving your IPFS data from Brave to [IPFS Desktop](https://github.com/ipfs/ipfs-desktop), ensuring you don't lose any of your important files, and keep access to IPFS resources in your browser.
```

## Why Migrate?

> - **Imminent Removal:** The IPFS node feature in Brave is being [phased out](https://github.com/brave/brave-browser/issues/37735#issuecomment-2247764368) and will happen once you update to v1.69.153 or later. To ensure uninterrupted access to your IPFS data, migration is necessary, especially if you pinned something, or published with IPNS.

Reviewer 2: Add a note to ensure users don't freak out about losing data
```suggestion
- **Imminent Removal:** The IPFS node feature in Brave is being [phased out](https://github.com/brave/brave-browser/issues/37735#issuecomment-2247764368) and will happen once you update to v1.69.153 or later. Although upgrading will not delete data associated with the IPFS node, migration is necessary to ensure uninterrupted access to your IPFS data, especially if you pinned something, or published with IPNS.
```

## Moving the Brave IPFS Repository

The IPFS repository, often referred to as `$IPFS_PATH` (aka `~/.ipfs`), contains all your IPFS data, IPNS keys, and PeerID identify of your IPFS node. Brave's IPFS Node used the same repository format as Kubo, making migration to IPFS Desktop relatively easy.

If you did not use IPFS Desktop before, you can simply swap `.ipfs` created by IPFS Desktop with the one from your Brave node. This is the simplest way of migrating your node, all data, pins, IPNS keys, addresses and PeerID will remain the same and IPNS publishing will continue working.

First, we need to locate your Brave IPFS repository. The configuration directory for the Brave managed IPFS node can be found in the browser’s profile directory in a subfolder named `brave_ipfs`. You can find your IPFS directory by opening `brave://version/`, finding "Profile Path", and replacing `/Default` with `/brave_ipfs`:

<!-- TODO: confirm these paths are valid -->
- Windows: `%LOCALAPPDATA%\BraveSoftware\Brave-Browser\User Data\brave_ipfs`
  - example: `C:\Users\YOURUSERNAME\AppData\Local\BraveSoftware\Brave-Browser\User Data\brave_ipfs`

> - macOS: `~/Library/Application Support/BraveSoftware/Brave-Browser/brave_ipfs`

Reviewer 2: Confirming this is correct

```

### Updating RPC URL in IPFS Companion

Brave used custom ports: `45001` for RPC and `48080` for Gateway.

If IPFS Companion browser extension does not detect your node after migrating repository from Brave, you need to update RPC and Gateway URLs in Companion preferences.

- Change the **Kubo RPC URL** from `http://127.0.0.1:5001` to `http://127.0.0.1:45001`
- Change the **Local Gateway** from `http://127.0.0.1:8080` to `http://127.0.0.1:48080`

Alternative is to update `.ipfs/config` and replace all occurences of `45001` with `5001` and `48080` with `8080`. Make sure you do not have anything listening on these ports before you make the change.

## Conclusion

Congratulations! You've successfully migrated your IPFS data from Brave to IPFS Desktop.

If you encountered any challenges during the migration process or need further assistance, please don't hesitate to leave a comment in the thread below. The community is here to help, and your feedback can also assist others who might be going through the same process.

## FAQ

### Is it possible to move `brave_ipfs` to a different location than `.ipfs`?

> Yes, but one needs to set `IPFS_PATH` environment variable before running IPFS Desktop to point at the new location.

Reviewer 2: ```suggestion
Yes, but you need to set the `IPFS_PATH` environment variable before running IPFS Desktop to point at the new location.
```

See [How does IPFS Desktop select the IPFS repo location?](https://github.com/ipfs/ipfs-desktop/?tab=readme-ov-file#how-does-ipfs-desktop-select-the-ipfs-repo-location)

### Where can I find FAQ/Troubleshooting for IPFS Desktop?

See [github.com/ipfs/ipfs-desktop/#faq--troubleshooting](https://github.com/ipfs/ipfs-desktop/?tab=readme-ov-file#faq--troubleshooting)

### Can Kubo be used instead?

Yes, advanced users who are comfortable with command-line can use [Kubo](https://docs.ipfs.tech/install/command-line/) instead of IPFS Desktop, and run it against a custom `IPFS_PATH` to run a headless daemon, or perform selective manual migration via CLI.

### How to export my Files (MFS) with Kubo CLI?

> To export contents of MFS to a CAR, one could:

Reviewer 2: ```suggestion
To export contents of MFS to a CAR, run the following commands:
```

---

> title: 'Migrating From Brave to Desktop App'

Reviewer: are you intentionally calling it "Desktop App" instead of IPFS Desktop? sounds kinda odd/generic (and also Brave is a desktop app)

description: ' Complete Guide to Migrating Your IPFS Data'
date: 2024-08-13
header_image: '/braveinfobar2.png'
tags:
- brave
- browsers
- IPFS Desktop
---

> If you've been using Brave browser's built-in IPFS node, you might be ready to

Reviewer: should be far more clear/transparent here to inform both communities about what is happening and why this is being posted.

specifically:

- state that brave is removing ipfs support
- link to the PR discussion
- state the version and release date

Reviewer 2: Here's a link to the PR: https://github.com/brave/brave-browser/issues/37735

The IPFS repository, often referred to as `$IPFS_PATH` (aka `.ipfs`), contains all your IPFS data.

If you did not use IPFS Desktop before, you can simply swap `.ipfs` created by IPFS Desktop with the one from your Brave node.
This is the simplest way of migrating your node, all data, pins, IPNS keys, addresses and PeerID will remain the same and IPNS publishing will continue working.

First, we need to locate your Brave IPFS repository:

<!-- TODO: confirm these paths are valid -->
- Windows: `%LOCALAPPDATA%\BraveSoftware\Brave-Browser\User Data\Default\ipfs`
- macOS: `~/Library/Application Support/BraveSoftware/Brave-Browser/Default/ipfs`
- Linux: `~/.config/BraveSoftware/Brave-Browser/Default/ipfs`

Now, we'll move this repository to the default location for IPFS Desktop:

- Windows: `%USERPROFILE%/.ipfs`
- macOS and Linux: `~/.ipfs`

To move the directory, you can use the following commands in your terminal or command prompt:

For Windows:

```
move "%LOCALAPPDATA%\BraveSoftware\Brave-Browser\User Data\Default\ipfs" "%USERPROFILE%\.ipfs"
```

For macOS and Linux:

```
mv ~/Library/Application\ Support/BraveSoftware/Brave-Browser/Default/ipfs ~/.ipfs
```

> <!-- TODO: decide if we even go here

Reviewer: Kinda wild that for a location-agnostic technology we don't have a nice tool for replication. Is there really no way to 1) get a list of root CIDs from my node A and 2) input that list into my other node B to fetch/

Author: You can mirror CIDs in MFS with two CLI commands. But there is a lot of UX beyond CID (IPNS keys, PeerID and ports, any configured remote pinning services etc). Each thing is multiple CLI commands to move (and break) so I'm thinking a power users will know them anyway, and regular users reading the blogpost should just move the profile.

> move "%LOCALAPPDATA%\BraveSoftware\Brave-Browser\User Data\Default\ipfs" "%USERPROFILE%\.ipfs"
> ```
> 
> For macOS and Linux:
> 
> ```
> mv ~/Library/Application\ Support/BraveSoftware/Brave-Browser/Default/ipfs ~/.ipfs
> ```

Author: ````suggestion
move "%LOCALAPPDATA%\BraveSoftware\Brave-Browser\User Data\Default\brave_ipfs" "%USERPROFILE%\.ipfs"
```

For macOS and Linux:

```
mv ~/Library/Application\ Support/BraveSoftware/Brave-Browser/Default/brave_ipfs ~/.ipfs
```
````
